recipes/icu/meta.yaml

```diff
@@ -25,6 +25,20 @@ requirements:
   build:
     - python
 
+test:
+  commands:
+    - derb --help             # [unix]
+    - genbrk --help           # [unix]
+    - gencfu --help           # [unix]
+    - gencnval --help         # [unix]
+    - gendict --help          # [unix]
+    - genrb --help            # [unix]
+    - icu-config --help       # [unix]
```

Comment:
Maybe the dash here causes it to fail. This is the command it dies on (before running it).